vscode_extension/package.json

```diff
@@ -30,6 +30,7 @@
     "onCommand:sorbet.setLogLevel",
     "onCommand:sorbet.showOutput",
     "onCommand:sorbet.toggleHighlightUntyped",
+    "onCommand:sorbet.configureHighlightUntyped",
```

Comment:
This list, and commands, is sorted alphabetically to make it easier to detect duplicates / conflicts (common issue is for ppl to define the same thing twice which throws a wrench when debugging).